packages/asc-ui/src/components/Button/ShareButton/ShareButton.tsx

```diff
@@ -6,20 +6,24 @@ import ShareButtonStyle, {
 
 type Props = {
   iconSize?: number
+  title?: string
```

Comment:
`title` is already part of `HTMLAttributes < HTMLButtonElement >`, so don't need it here